src/ExportGenerators.mss

```diff
@@ -984,6 +979,12 @@ function GenerateRest (bobj) {
         libmei.AddAttribute(r, 'color', nrest_color);
     }
 
+    position = bobj.RestPosition;
+    if (position != 0 and name != 'space')
+    }
+	libmei.AddAttribute(r, 'loc', (position + 64) / 16);
+    }
```

Comment:
You have a wrong bracket and a wrong indentation here.

Reply:
I hate this. To make sure I have the correct indentation, I copied another line …